test/Rules/BUY_PROPERTY.test.js

```diff
@@ -81,5 +88,39 @@ describe('Rules -> BUY_PROPERTY', () => {
         `Property ownership was not updated after ${inputEvent} event`
       );
     });
+    it('should make a call to the UI#playerShortOnFunds when funds are insufficient', () => {
+      const uiSpy = sinon.spy();
+      userInterface.playerShortOnFunds = uiSpy;
+      const property =
+        gameState.config.propertyConfig.properties[TEST_PROPERTY];
+      gameState.currentBoardProperty = property; // price: 60
+      gameState.currentPlayer.cash = 50;
```

Comment:
Should/Could we do:
```javascript
gameState.currentPlayer.cash = property.price - 1; // or 10
```
It would remove magic numbers (tbh, I don't think a `1` really counts ? That's a tough question haha)